--- sc68-2.2.1/Makefile.am.orig	2003-09-25 22:03:42.000000000 +0200
+++ sc68-2.2.1/Makefile.am	2023-06-23 20:16:46.019407948 +0200
@@ -51,7 +51,6 @@ my_install_headers =\
  ./io68/ym_io.h\
  ./io68/ymemul.h\
  ./io68/ymoutorg.h\
- ./unice68/unice68.h\
  ./config68.h\
  ./config_option68.h\
  ./config_package68.h\
--- sc68-2.2.1/configure.ac.orig	2003-09-30 11:58:46.000000000 +0200
+++ sc68-2.2.1/configure.ac	2023-06-23 20:17:11.545936326 +0200
@@ -68,7 +68,7 @@ AC_SUBST(REPLAY_RATE_MIN)
 # #
 
 # Libraries
-SUBDIRS68="emu68 emu68debug io68 desa68 unice68 file68 api68"
+SUBDIRS68="emu68 emu68debug io68 desa68 file68 api68"
 # Programs
 SUBDIRS68="$SUBDIRS68 as68 sc68 info68 sourcer68 debug68"
 # Data
@@ -260,7 +260,6 @@ AC_CONFIG_FILES([
  info68/Makefile
  debug68/Makefile
  sourcer68/Makefile
- unice68/Makefile
  man/Makefile
  info/Makefile
  doc/Makefile
--- sc68-2.2.1/debug68/Makefile.am.orig	2003-09-25 21:55:21.000000000 +0200
+++ sc68-2.2.1/debug68/Makefile.am	2023-06-23 20:17:58.522348500 +0200
@@ -18,7 +18,7 @@ bin_PROGRAMS = debug68
 debug68_SOURCES = $(mysources) $(myheaders)
 debug68_LDADD = \
  ../file68/libfile68.la\
- ../unice68/libunice68.la\
+ -lunice68\
  ../desa68/libdesa68.la\
  ../emu68debug/libemu68dbg.la
 debug68_CFLAGS = -DEMU68DEBUG=1 ${AM_CFLAGS}
--- sc68-2.2.1/info68/Makefile.am.orig	2003-09-25 21:56:24.000000000 +0200
+++ sc68-2.2.1/info68/Makefile.am	2023-06-23 20:18:26.538863388 +0200
@@ -1,4 +1,4 @@
 bin_PROGRAMS = info68
 info68_SOURCES = info68.c info68_doc.h
-info68_LDADD = ../file68/libfile68.la ../unice68/libunice68.la
+info68_LDADD = ../file68/libfile68.la -lunice68
 #EXTRA_DIST = info68.vcproj
--- sc68-2.2.1/sc68/Makefile.am.orig	2003-09-25 21:56:36.000000000 +0200
+++ sc68-2.2.1/sc68/Makefile.am	2023-06-23 20:19:18.851913317 +0200
@@ -3,7 +3,7 @@ sc68_SOURCES = sc68.c sc68_doc.h
 sc68_LDADD =\
  ../api68/libapi68.la\
  ../file68/libfile68.la\
- ../unice68/libunice68.la\
+ -lunice68\
  ../io68/libio68.la\
  ../emu68/libemu68.la
 
--- sc68-2.2.1/sourcer68/Makefile.am.orig	2003-09-25 21:56:40.000000000 +0200
+++ sc68-2.2.1/sourcer68/Makefile.am	2023-06-23 20:19:40.945126961 +0200
@@ -2,7 +2,7 @@ bin_PROGRAMS = sourcer68
 sourcer68_SOURCES = sourcer68.c sourcer68_doc.h
 sourcer68_LDADD = \
  ../file68/libfile68.la\
- ../unice68/libunice68.la\
+ -lunice68\
  ../desa68/libdesa68.la\
  ../emu68debug/libemu68dbg.la
 sourcer68_CFLAGS =\
--- sc68-2.2.1/file68/ice68.c.orig	2003-09-22 17:20:34.000000000 +0200
+++ sc68-2.2.1/file68/ice68.c	2023-06-23 20:57:40.052779975 +0200
@@ -24,11 +24,11 @@
 #include "file68/error68.h"
 #include "file68/alloc68.h"
 #include "file68/istream68_file.h"
-#include "unice68/unice68.h"
+#include <unice68.h>
 
 int ice_is_magic(const void * buffer)
 {
-  return unice68_get_depacked_size(buffer, 0) > 0;
+  return unice68_depacked_size(buffer, 0) > 0;
 }
 
 void *ice_load(istream_t *is, int *ulen)
@@ -45,7 +45,7 @@ void *ice_load(istream_t *is, int *ulen)
   }
 
   csize = 0;
-  dsize = unice68_get_depacked_size(header, &csize);
+  dsize = unice68_depacked_size(header, &csize);
 
   if (dsize < 0) {
     SC68error_add("ice_load(%s) : not ICE!", fname);
